### DISCUSSION TITLE
Hide 'Snapchat won't run without Google Play services.....'

### DISCUSSION THREAD

**Discussion by ghost** [rodit/SnapMod#112] [Ideas & Feature Suggestions]
When Snapchat is ran on a phone without Google Play services you get an annoying notification that says 'Snapchat won't run without Google Play services, which are not support by your device.' Even though it works perfectly fine without them.

**Comment by rodit** [comment]
If this is easy and takes under 10 minutes I'll do it but I can imagine this affects less than 0.01% of users so I will not invest much time into it (not to mention I can't really test it).

**Comment by Magna-Graecia** [comment]
There is an Xposed module made for this issue. Try using this: https://forum.xda-developers.com/t/mod-xposed-no-more-google-play-services-2017-02-08-v1-1.3552827/

**Reply by ghost** [reply]
Works great. Thank you!

**Reply by Magna-Graecia** [reply]
You're welcome! Glad it helped :)
